### ISSUE TITLE
https://raku.org/compilers/std-viv

### ISSUE THREAD

**Issue by coke** [Raku/raku.org#211]
https://raku.org/compilers/std-viv is outdated and we should consider removing it, or at least updating the text with more than the warning box.

**Comment by 2colours** [comment]
Right now, it's
![image](https://github.com/Raku/raku.org/assets/37218286/6dc21a28-8e3a-4949-8895-5c1731ddf329)

**Comment by 2colours** [comment]
Yes, perhaps. good for demo purposes but perhaps the server will need some extra config, similar to the doc site.

**Comment by coke** [comment]
I think @librasteve has removed this in the reworked site.

**Comment by librasteve** [comment]
that's correct - can close this issue
